### PR TITLE
Feature/quit warning

### PR DESCRIPTION
**Pull request by TimothyTichelaar**
Made a "Pop-up" screen to prevent accidentally quitting the game #SaveDesign enzo

### CODE REVIEW

File: Assets/Scripts/Controllers/Menu/PLaytest/Outro.cs
Comment (hexaclue) [inline Assets/Scripts/Controllers/Menu/PLaytest/Outro.cs:1]: Should rename: capital letter typo in folder name "PLaytest" --> "Playtest"

**Review by hexaclue — CHANGES_REQUESTED** [review]
lgtm, fix required

File: Assets/Scripts/Controllers/Menu/PLaytest/Outro.cs
Comment (TimothyTichelaar) [reply Assets/Scripts/Controllers/Menu/PLaytest/Outro.cs:1]: Renamed the folder and another file that had the same typo

**Comment by hexaclue** [comment]
Does not work as intended

quit

**Comment by TimothyTichelaar** [comment]
Worked for me, but changed the sort order to always display on top of the other menu. Should work without issues now

File: Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        _yesButton = _root.Q<Button>("YesButton");
+        _yesButton.clicked += OnYesButton;
+        _noButton = _root.Q<Button>("NoButton");
+        _noButton.clicked += OnNoButton;
+    }
+
+    void OnYesButton(){
+        Logger.Log("MenuController", "THE MENU IS DEAD!");
+		//Application.Quit();
```
Comment (hexaclue) [inline Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs]: Why is this commented? The user can click the yes button but nothing would happen 😅

Comment (TimothyTichelaar) [reply Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs]: This was Wouter, Application.Quit() doesnt work when testing in unity thats why it is logged to see if it is triggerd

Comment (hexaclue) [reply Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs]: I'd say uncomment the line with the app quit so it works when in a built state 😁 

hbu @Justwouter ?

Comment (Justwouter) [reply Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs]: Already fixed on my branch 😅
Will add it here as well.

**Review by Justwouter — APPROVED** [review]
Just press atl+shift+f once in a while please :')
LGTM tho

**Review by hexaclue — CHANGES_REQUESTED** [review]
All issues from before have been fixed. ✅ 

⚠️ The quit warning box buttons don't respond to mouseover/hover. (neither do the ones on the pause menu box, but I suppose that one's irrelevant for this PR.

**Comment by hexaclue** [comment]
This too does not work, sadly

**Comment by hexaclue** [comment]
Does not work as intended

**Comment by Justwouter** [comment]
@TimothyTichelaar  Weet niet precies wat er verkeerd gaat met de UXML/USS styling maar maybe use callback events als er geen oplossing te vinden is?
 
Dit werkt iig:
```
 _quit.RegisterCallback<MouseOverEvent>((type) => {
	_quit.style.unityBackgroundImageTintColor = new Color(0, 0, 0, 1f);
 }); 
 _quit.RegisterCallback<MouseOutEvent>((type) => {
	_quit.style.unityBackgroundImageTintColor = new Color(255, 255, 255, 1f);
 });
```

Lijkt te maken te hebben met templates. Options & keybinds hebben templates en werken wel terwijl quit menu en pause menu zonder niet werken.

Never mind. As it turns out, the styling was inline set to be the darker selection color but because inline styling overwrites stylesheets the UI was never updated... I hate frontend

**Comment by Justwouter** [comment]
@TimothyTichelaar @hexaclue this good?

**Review by hexaclue — APPROVED** [review]
LGTM.

**Pull request merged**